input/fsh/EX_Basic.fsh

```diff
@@ -350,21 +350,21 @@ Description: "Hemoglobin lab result to support TNMStageGroup example"
 * performer = Reference(us-core-practitioner-owen-oncologist)
 * valueQuantity = 13.5 'g/dl' "g/dl"
 
-Instance: history-metastatic-cancer-jenny-m 
```

Comment:
Change "jenny-m" because she's the extended example patient. This is in the group of basic examples.